pinot-core/src/test/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTrackerTest.java

```diff
@@ -85,29 +85,37 @@ public void testRecordIngestionDelayWithNoAging() {
 
     // Test we follow a single partition up and down
     for (long i = 0; i <= maxTestDelay; i++) {
-      ingestionDelayTracker.updateIngestionDelay(i, partition0);
+      ingestionDelayTracker.updateIngestionDelay(i, (i + 1), partition0);
```

Comment:
Parentheses are not needed in `(i + 1)`

Reply:
I think these enhance readability: want to highlight we are using a couple of values that are different